person.rb

```diff
@@ -1,13 +1,15 @@
 require './module/nameable'
+require './module/rental'
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- Please ensure a method for adding rentals should be present for your person class, similar to this 👇🏽
```
`def add_rental(book, date)`
### *`The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.`
`Rental.new(date, book, self)`
`end`
```

Reply:
@rotimiazeez 1 person can have many rentals and 1 rental can only be owned by one person. This method is implemented on the `rentals.rb`.  And all the rentals related to each individual is stored in an instance variable array : `rentals` So the above-implemented change is on the `rentals.rb`. I suggest you recheck it and any changes requested will be appreciated. :smiley:

Reply:
Hi @SimonGideon,

I can see the method is implemented in the rentals(guess I missed it) 🤦🏽‍♂️. You can add the parameters for many to many association as suggested 😉  I think it will make the code more readable. 👍🏼

Reply:
@rotimiazeez :clap: Noted. The changes are implemented on the future commits. :muscle: